easybuild/easyconfigs/p/protobuf/protobuf-2.5.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'protobuf'
+version = '2.5.0'
+
+homepage = 'https://github.com/google/protobuf/'
+description = """Google Protocol Buffers"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['https://github.com/google/protobuf/archive/v%(version)s/']
```

Comment:
The checksum of the archive from this URL is different from the one downloaded from `https://github.com/google/%(namelower)s/releases/download/v%(version)s/`